client/my-sites/people/followers-list/followers.jsx

```diff
@@ -220,4 +224,4 @@ class Followers extends Component {
 	}
 }
 
-export default localize( Followers );
+export default connect( null, { recordGoogleEvent } )( localize( Followers ) );
```

Comment:
It's bit of a bummer that we're reintroducing `connect` right after we removed it 🙂

Reply:
Agreed. I already made a note to revisit this component for cleanup/refactoring possibilites after the reduxification is over. I see a potential function component which could use a simpler way of dispatching actions. In general analytics seem like a good use case for Redux to me. – do you see any alternatives to that approach @jsnajdr?

Reply:
I agree it's OK to keep Redux here. Refactoring to function component and `useDispatch` should simplify the usage.